<launch>
    <machine name="local" address="localhost" default="true" />

      <arg name="node_start_delay" default="5.0" />

      <node name="rviz_first_map" pkg="rviz" type="rviz" respawn="true"
            args="-d $(find slam_constructor)/rviz/debug.rviz" />

      <node name="rviz_second_map" pkg="rviz" type="rviz" respawn="true"
            args="-d $(find slam_constructor)/rviz/debug_second_map.rviz" />
          
      <node name="rviz_conj" pkg="rviz" type="rviz" respawn="true"
            args="-d $(find slam_constructor)/rviz/debug_conj.rviz" />
          
      <node name="rviz_disj" pkg="rviz" type="rviz" respawn="true"
            args="-d $(find slam_constructor)/rviz/debug_disj.rviz" />

      <node pkg="slam_constructor" type="conj_and_disj" name="conj_and_disj" output="screen"
          launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' "/>
</launch>